### PR TITLE
Remove overheads and update dependencies

### PR DESCRIPTION
**Pull request by GitBruno**
- Remove `tidy-markdown` (tidy markdown ourselves)
- Remove `xo` (Too much overhead for a small project)
- tidy package.json

We are alive again!

### CODE REVIEW

**Comment by Hypercubed** [comment]
Are we dead?

Thanks for the PR... this is a lot of changes to one PR and not sure I want to accept all of them.  I'll try to merge soon peice by peice.

**Comment by GitBruno** [comment]
Thanks! I can split them up if you like. (I checked out the dev-branch, I can re-do this with Master and split specifics).

This package was not working on the latest version of Node for me. (From memory, tidy-markdown failed.)

So there are only two things, rest can be ignored:
  1. Removed `tidy-markdown` as a dependency (This might be good or bad, but needs work either way)
  2. Added fencing option

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
+.DS_Store
 node_modules
+package-lock.json
```
Comment (GitBruno) [inline .gitignore:3]: Maybe you want this file? but it was not in the original package.

File: .travis.yml [hunk 2]
```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
   - "4"
   - "6"
-  - "0.10"
```
Comment (GitBruno) [inline .travis.yml:5]: Updated dependencies don't run in this version.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +0,0 @@
...
-
-## 1.2.1 (2016-06-15)
-_(none)_
-
-## 1.2.0 (2016-06-14)
-
---------------------
-* Moved markdown formmater from tap-summary to tap-markdown
-
-undefined
```
Comment (GitBruno) [inline CHANGELOG.md:17]: I removed this file as it seems not be in use?

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Jayson Harshbarger
```
Comment (GitBruno) [inline LICENSE:3]: From dev-branch and readme, it is probably a good idea to choose one or the other.

File: README.md [hunk 5]
```diff
@@ -8,7 +8,10 @@ Consume TAP, produce markdown.
 
 ## Install
 
-npm install tap-markdown -g
```
Comment (GitBruno) [inline README.md:11]: We should not need to install globally

File: README.md [hunk 6]
```diff
@@ -8,7 +8,10 @@ Consume TAP, produce markdown.
...
+    npm install tap-markdown
+
+> If you must support Node < 4 you can explicitly install older version of tap-markdown by running the following `npm install tap-markdown@1.2.1`
```
Comment (GitBruno) [inline README.md:13]: This is a node reflects where we are in light of the updated dependencies.

File: README.md [hunk 7]
```diff
@@ -21,7 +24,7 @@ or in `package.json`
 ```json
 {
   "scripts": {
-    "test": "tape test/*.js | tap-markdown"
+    "test": "tape test/*.js | tap-markdown > results.md"
```
Comment (GitBruno) [inline README.md:27]: Add a sample that shows you how to write out to a file.

File: README.md [hunk 8]
```diff
@@ -43,15 +46,17 @@ or in `package.json`
...
-### CLI Options
+### CLI Options	
+	
+```	
+  --duration       Disable duration
+  --fence          Enable fencing      
+```	
 
-```
-  --no-tidy        Disable tidy-markdown
```
Comment (GitBruno) [inline README.md:49]: Maybe we should have this option still?

File: README.md [hunk 9]
```diff
@@ -43,15 +46,17 @@ or in `package.json`
 - fail: 0
 ```
 
-### CLI Options
+### CLI Options	
+	
+```	
+  --duration       Disable duration
+  --fence          Enable fencing
```
Comment (GitBruno) [inline README.md:53]: Enable to make it backward compatible?

File: test/fixtures/comment.expected [hunk 10]
```diff
@@ -23,3 +23,4 @@ Hello,
 ```
 world!
 ```
+
```
Comment (GitBruno) [inline test/fixtures/comment.expected:26]: This is annoying, not sure if there is an elegant way of fixing this?

File: index.js [hunk 11]
```diff
@@ -57,32 +63,27 @@ Formatter.prototype.summary = function (summary) {
 
 Formatter.prototype.comment = function (comments) {
   var output = [LF];
-  output.push('# Comments');
-  // output.push('```');  TODO: make this an option?
+  output.push('# Comments' + LF);
   output.push(Object.keys(comments).map(function (name) {
-    return '## ' + name + LF + comments[name].join(LF);
+    return '## ' + name + LF + LF + fence + LF + comments[name].join(LF).replace('    ','') + LF + fence;
```
Comment (GitBruno) [inline index.js:68]: Should replace contain a smarter regex?

**Review by GitBruno — COMMENTED** [review]
Hey, thank you so much for looking at this. I thought I leave some comments on the changes. Hope that is helpful. Again, if you want me to create a bunch of smaller PRs just let me know.

**Comment by GitBruno** [comment]
Hey Jayson, how are you going with this? I want to publish a module with this package as a dependency. If you don't have time for this that is OK, I'll find another way to include a copy.

**Comment by GitBruno** [comment]
@Hypercubed Is there anything I can do to help?

**Comment by Hypercubed** [comment]
Sorry, I haven't had time to look.  It's hard to review code for a project I haven't actively used in a while.

**Pull request closed without merging**